docs/docs/miscellaneous/chart-params.mdx

```diff
@@ -18,7 +18,7 @@ Note not all fields are correctly categorized. The fields vary based on visualiz
 | Field             | Type     | Notes                               |
 | ----------------- | -------- | ----------------------------------- |
 | `database_name`   | _string_ | _Deprecated?_                       |
-| `datasource`      | _string_ | `<datasouce_id>__<datasource_type>` |
+| `datasource`      | _string_ | `<datasource_id>__<datasource_type>` |
```

Comment:
I'm happy to update the table, but I'd rather wait until I get a general acceptance of the changes.

Reply:
I guess we're there now :D